styles.css

```diff
@@ -244,9 +251,9 @@ a {
     bottom: 0;
     left: 0;
     background-color: var(--black);
-    -webkit-transition: width 0.3s ease-in-out;
-    -o-transition: width 0.3s ease-in-out;
-    transition: width 0.3s ease-in-out;
+    -webkit-transition: 0.3s ease-in-out;
+    -o-transition: 0.3s ease-in-out;
+    transition: 0.3s ease-in-out;
```

Comment:
Now you used transition without animated property declaration. It means definitely `all` properties for transition. So, now you're animating not only `width`, but all properties that changes.